samples/web/content/apprtc/js/appcontroller.js

```diff
@@ -134,6 +146,11 @@ var AppController = function(loadingParams) {
       this.roomSelection_.onRoomSelected = function(roomName) {
         this.hide_(roomSelectionDiv);
         this.finishCallSetup_(roomName);
+
+        this.roomSelection_ = null;
+        if (this.localStream_) {
```

Comment:
I don't think we want to ask for the camera if we're not going to display it right away.